pkg/operation/botanist/component/etcd/crds/templates/crd-druid.gardener.cloud_etcds.yaml

```
@@ -0,0 +1,1719 @@
+                              type: string
+                            type: array
+                        required:
+                          - key
+                          - operator
+                        type: object
+                      type: array
+                    matchLabels:
+                      additionalProperties:
+                        type: string
+                      description: matchLabels is a map of {key,value} pairs. A single
+                        {key,value} in the matchLabels map is equivalent to an element
+                        of matchExpressions, whose key field is "key", the operator
+                        is "In", and the values array contains only "value". The requirements
+                        are ANDed.
+                      type: object
+                  type: object
+                lastError:
+                  description: LastError represents the last occurred error.
+                  type: string
+                members:
+                  description: Members represents the members of the etcd cluster
+                  items:
+                    description: EtcdMemberStatus holds information about a etcd cluster
+                      membership.
+                    properties:
+                      id:
+                        description: ID is the ID of the etcd member.
+                        type: string
+                      lastTransitionTime:
+                        description: LastTransitionTime is the last time the condition's
+                          status changed.
+                        format: date-time
+                        type: string
+                      name:
+                        description: Name is the name of the etcd member. It is the
+                          name of the backing ` + "`Pod`" + `.
+                        type: string
+                      reason:
+                        description: The reason for the condition's last transition.
+                        type: string
+                      role:
+                        description: Role is the role in the etcd cluster, either ` + "`Leader`" + `
+                          or ` + "`Member`" + `.
+                        type: string
+                      status:
+                        description: Status of the condition, one of True, False, Unknown.
+                        type: string
+                    required:
+                      - lastTransitionTime
+                      - name
+                      - reason
+                      - status
+                    type: object
+                  type: array
+                observedGeneration:
+                  description: ObservedGeneration is the most recent generation observed
+                    for this resource.
+                  format: int64
+                  type: integer
+                ready:
+                  description: Ready represents the readiness of the etcd resource.
```

```suggestion
                  description: Ready is `true` if all etcd replicas are ready.
```
[Ref](https://github.com/gardener/etcd-druid/blob/53db11273e0636cdef23891942e289bbb4aa7f6a/config/crd/bases/10-crd-druid.gardener.cloud_etcds.yaml#L1692), if you're already on it 🙂